libs/designsystem/item/src/item.component.ts

```diff
@@ -57,9 +70,18 @@ export class ItemComponent {
     }
   }
 
+  /** Focus the internal button of the ion-item */
+  focusItem(item: HTMLIonItemElement) {
```

Comment:
If focus is placed on the kirby-item, the focus is proxied down to the Ionic button within the shadowroot of the ion-item. However the ion-item needs to have [button]=true for this to happen which it can't have if it contains a checkbox or radio. When the kirby-item is within a menu it needs to have [button]="true" for keyboard interaction to be enabled with the current solution.